web/blog/vast-v2.2/index.md

```diff
@@ -96,3 +96,19 @@ summarize:
 In SQL, this would be the `AS` token: `SELECT min(ts) AS min_ts`.
 
 [summarize]: /docs/understand-vast/query-language/operators/summarize
+
+## Regular Expression Predicates
+
+The light at the end of the tunnel for regular expression support is here. VAST
+can now execute query expressions with predicates of the form `x ~ /f.o.*bar/`.
+
+We introduced this functionality "accidentally" during a refactoring that
+transposed predicate evaluation from being row-wise to column-wise. That said,
+the performance is not anywhere near where we would deem it production-grade. We
+have planned a series of optimizations for the next coming releases. Stay tuned!
```

Comment:
This still causes warnings in the evaluator (fixed by #2487), but that's not going to be part of this release. I would not want to announce this now, especially since we're currently instantiating the regex object once per event instead of once per batch or ideally once per query.

Reply:
Fair enough, I'm in fact also for _not_ announcing it. But then I thought there might be value for those folks that have just small amounts of testing data.

Feel free to scratch the block.

Reply:
For what it's worth, by reading
`(dport <= 1024 || :addr in 10.0.0.0/8) && ! (#type == /zeek.*/)` in https://vast.io/docs/understand-vast/query-language/expressions, I thought it was already supported and was a bit confused about the warning I got on the server.

As alternative, could be announced as a alpha/beta feature with disclaimer about performance and warnings :-). It would give some lead time for users to integrate with it.

Reply:
We'll do a pass over the docs page for expressions soon. For now, I've removed the section from the release blog post.